packages/currency/src/native.ts

```diff
@@ -75,6 +75,12 @@ export const nativeCurrencies: Record<NativeCurrencyType, (NativeCurrency & { na
       name: 'Near Testnet',
       network: 'aurora-testnet',
     },
+    {
+        symbol: 'ARETH',
```

Comment:
we usually put testnet in the symbol as well so we can differentiate between testnet and mainnet, like on Near

Reply:
Had no idea, congrats on the first PR though  :)